benchmark/url/legacy-vs-whatwg-url-searchparams-parse.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+const common = require('../common.js');
+const { URLSearchParams } = require('url');
+const querystring = require('querystring');
+
+const inputs = {
+  noencode: 'foo=bar&baz=quux&xyzzy=thud',
+  encodemany: '%66%6F%6F=bar&%62%61%7A=quux&xyzzy=%74h%75d',
```

Comment:
`URLSearchParams` doesn't support custom separators, so `multicharsep` is not needed.

I copied the input from url-searchparams-* benchmarks, looks like there isn't `encodefake` in them.